server/controllers/auth.controller.js

```diff
@@ -164,8 +165,10 @@ function loginWithEmail(req, res, next) {
  */
 
 function register(req, res, next) {
+
   const { username } = req.body;
   const { password } = req.body;
+  const newsletterSignup = req.body.newsletter;
```

Comment:
Wondering if we should modify the user.model to set a `signedupForNewsletter` bool just for future reference.